### ISSUE TITLE
Exodii Quadruped's tile is of Bipedal entity

### ISSUE THREAD

**Issue by JinJanODWan** [SomeDeadGuy/UndeadPeopleTileset#79]
In [normal_monster.png](https://github.com/SomeDeadGuy/UndeadPeopleTileset/blob/master/TILESETS/gfx/MSX%2B%2BUnDeadPeopleEdition/normal_monsters.png) entity in row 3 column 11 is _bipedal_ (moving on two feet/legs) while the [description and name](https://github.com/CleverRaven/Cataclysm-DDA/blob/864c53362701936cf0a34697d9d2dee6f7c8b851/data/json/monsters/cyborgs.json#L125-L126) of said entity ("Exodii quadruped: This enormous quadrupedal robot seems to be cobbled together..") tells us it moves on four feet/legs.

Any chance for an update so the tile is more true to the description?

**Comment by SomeDeadGuy** [comment]
Yeah, those robot sprites were in the game before. Whole exodii creatures need to be made from scratch.
